systemtest/src/main/java/io/strimzi/systemtest/security/OpenSsl.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.security;
+
+import io.strimzi.systemtest.Constants;
+import io.strimzi.test.TestUtils;
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class OpenSsl {
```

Comment:
Adding a JavaDoc to this class would be great :) explaining the main purpose of it and what is provides...

Reply:
Will do. Thanks!